Identity Engine Updates - Post Tuscany: Embedded SDK Use-Case Self-Reg re-org (#2517)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/nodejs/summaryofsteps.md

The following sequence diagrams show the sequence of steps for self-service registration:

> ### Start new user registration with password factor

Reviewer: ```suggestion
### Start a new user registration with the password authenticator
```

</div>

> ### Enroll and verify email factor

Reviewer: ```suggestion
### Enroll and verify the email authenticator
```

</div>

> ### Option 1: Skip optional remaining factors

Reviewer: ```suggestion
### Option 1: Skip the optional remaining authenticators
```

The user can also opt to skip the factors when all of the remaining
factors are optional. In this case, the user opts to skip the phone
(SMS) factor.

<div class="common-image-format">

![Displays the Self-service registration skip phone flow diagram.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-seq-nodejs-skip-phone.png)

</div>

> ### Option 2: Enroll and verify phone (SMS) factor

Reviewer: ```suggestion
### Option 2: Enroll and verify the phone (SMS) authenticator
```

After the password and email are verified, the user has the option to
enroll in the phone factor.

> **Note:** Based on the steps described in [Set up your Okta org for a multifactor use case](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#set-up-your-okta-org-for-a-multifactor-use-case), the Okta application is set up to require one possession factor (either email or phone). After the email factor is verified, the phone factor becomes optional.

The following flow describes the steps when the user enrolls in the optional phone SMS factor.

<div class="common-image-format">

> ![Displays the Self-service registration enroll flow diagram.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-seq-nodejs-phone.png)

Reviewer: ```suggestion
![Displays the self-service registration enroll flow diagram.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-seq-nodejs-phone.png)
```
